src/styles/index.scss

```diff
@@ -14,7 +14,7 @@ body {
   min-height: 36px;
 }
 
-.notification.hidden {
+.hidden {
```

Comment:
The class `.hidden` might be too generic and could cause conflicts with other components. Consider changing it to a more specific name, such as `.notification-hidden`.